ccd-definition/CaseTypeTab.json

```diff
@@ -14,18 +14,28 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "Channel": "CaseWorker",
     "TabID": "HearingTab",
-    "TabLabel": "Hearing",
-    "TabDisplayOrder": 2,
+    "TabLabel": "Hearings",
+    "TabDisplayOrder": 1,
     "CaseFieldID": "hearing",
     "TabFieldDisplayOrder": 1
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "HearingTab",
+    "TabLabel": "Hearings",
+    "TabDisplayOrder": 2,
+    "CaseFieldID": "hearingDetails",
+    "TabFieldDisplayOrder": 1
+  },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "Channel": "CaseWorker",
     "TabID": "OrdersTab",
     "TabLabel": "Orders",
-    "TabDisplayOrder": 3,
```

Comment:
Why order 1? Isn't it th same as hearings above? Please clean up this file.

Reply:
👍 already changed

Reply:
It's weird now - it has got TabDisplayOrder: 1
as History and and Hearings, which means we are not explicit enough how we want to have this displayed.

Reply:
Looks ok now.